src/optimizer/operators.py

```diff
@@ -38,6 +39,7 @@ class OperatorType(IntEnum):
     LOGICALQUERYDERIVEDGET = 8,
     LOGICALUNION = 9,
     LOGICALORDERBY = 10
+    LOGICALLIMIT = 11
```

Comment:
Could we change this enum assignment to `auto()` like in the optimizer?

Reply:
Sure, I will do that.